redhat-appstudio/e2e-tests
# test: RHDP-513 add pipelinerun logs

# Description

Added pipelineRun logs after a pipeline is executed as failed to make these logs accessible.

## Issue ticket number and link
[RHDP-513](https://issues.redhat.com/browse/RHDP-513)

## Type of change


# How Has This Been Tested?
(You can see how it looks like in commit`[RHDP-513] add failed pipelinerun logs`. Go to `artifacts/redhat-appstudio-e2e/redhat-appstudio-e2e/build-log.txt` or `artifacts/redhat-appstudio-e2e/redhat-appstudio-e2e/artifacts/rp_preproc/results/xunit.xml` and search for `Logs from failed container`)

1. Edit `tests/e2e-demos/config/default.yaml` with a test that will make the pipelineRun fail. As an example:

```
tests:
  - name: "RHDP-466: create an application with nodejs component"
    applicationName: "e2e-nodejs"
    components:
      - name: "nodejs-component"
        type: "public"
        gitSourceUrl: "https://github.com/rsoaresd/simple-nodejs-app"
        devfileSource: "https://raw.githubusercontent.com/rsoaresd/simple-nodejs-app/main/devfiles/devfile.yaml"
        language: "nodejssss"
        healthz: "/"
        spec:
          replicas: 2
```
2. Run `make local/test/e2e`

# Checklist:

- [ ] I have added meaningful description with JIRA/GitHub issue key(if applicable), for example HASSuiteDescribe("[test_id:01] [DEVHAS-62](https://issues.redhat.com//browse/DEVHAS-62) devfile source") 
- [ ] I have updated labels (if needed)
